interceptor.go

```diff
@@ -141,6 +141,7 @@ func wrap(w http.ResponseWriter, r *http.Request, tx types.Transaction) (
 			} else if it != nil {
 				code := obtainStatusCodeFromInterruptionOrDefault(it, i.statusCode)
 				i.overrideWriteHeader(code)
+				i.w.Header().Del("Content-Length")
```

Comment:
If an interruption is raised processing the response body I think we should remove the `Content-Length` header because:
- It is still kind of a leaked information
- The connection is not gracefully closed (E.g. `transfer closed with 721 bytes remaining to read`) (which it was leading also to problems with the coraza e2e)